dom.bs

```diff
@@ -1413,6 +1413,152 @@ can only be used to influence an ongoing one.
+{{AbortSignal}} object). When an {{AbortController}} object is created, its
+<a for=AbortController>signal</a> must be set to a new {{AbortSignal}} object.
+
+<p>The <dfn attribute for=AbortController><code>signal</code></dfn> attribute's getter must return
+<a>context object</a>'s <a for=AbortController>signal</a>.
+
+<p>The <dfn method for=AbortController><code>abort()</code></dfn> method, when invoked, must
+<a for=AbortSignal>signal abort</a> on <a>context object</a>'s <a for=AbortController>signal</a>.
+
+
+<h3 id=interface-AbortSignal>Interface {{AbortSignal}}</h3>
+
+<pre class="idl">
+[Exposed=(Window,Worker)]
+interface AbortSignal : EventTarget {
+  readonly attribute boolean aborted;
+
+  attribute EventHandler onabort;
+};</pre>
+
+<dl class=domintro>
+ <dt><code><var>signal</var> . </code>{{AbortSignal/aborted}}
+ <dd>Returns true if this {{AbortSignal}} has been aborted, and false otherwise.
+</dl>
+
+<p>An {{AbortSignal}} object has an associated <dfn for=AbortSignal>aborted flag</dfn>. It is unset
+unless specified otherwise.
+
+<p>An {{AbortSignal}} object has associated <dfn for=AbortSignal>abort algorithms</dfn>, which is a
+<a for=/>set</a> of algorithms which are to be executed when its [=AbortSignal/aborted flag=] is
+set. Unless specified otherwise, its value is the empty set.
+
+<p class="note no-backref">The [=AbortSignal/abort algorithms=] enable APIs with complex
+requirements to react in a reasonable way to {{AbortController/abort()}}. For example, a given API's
+[=AbortSignal/aborted flag=] might need to be propagated to a cross-thread environment, such as a
+service worker.
+
+<p>The <dfn attribute for=AbortSignal>aborted</dfn> attribute's getter must return true if
+<a>context object</a>'s [=AbortSignal/aborted flag=] is set, and false otherwise.
+
+<p class=note>Changes to an {{AbortSignal}} object represent the wishes of the corresponding
+{{AbortController}} object, but an API observing the {{AbortSignal}} object can chose to ignore
+them. For instance, if the operation has already completed.
+
+<p>To <dfn export for=AbortSignal>signal abort</dfn>, given a {{AbortSignal}} object
+<var>signal</var>, run these steps:
+
+<ol>
+ <li><p>If <var>signal</var>'s [=AbortSignal/aborted flag=] is set, then return.
+
+ <li><p>Set <var>signal</var>'s [=AbortSignal/aborted flag=].
+
+ <li><p><a for=set>For each</a> <var>algorithm</var> in <var>signal</var>'s
+ [=AbortSignal/abort algorithms=]: run <var>algorithm</var>.
+
+ <li><p>[=Fire an event=] named <code event for=AbortSignal>abort</code> at <var>signal</var>.
+</ol>
+
+
+<h3 id=abortcontroller-api-integration>Using {{AbortController}} and {{AbortSignal}} objects in
+APIs</h3>
+
+<p>Any web platform API using promises that to represent operations that can be aborted must accept
+{{AbortSignal}} objects, use the [=AbortSignal/abort algorithms=] mechanism, and convey that the
+operation got aborted by rejecting the promise with the "{{AbortError}}" {{DOMException}}.
+
```

Comment:
It might help to also suggest how the signal is accepted, so that we don't sometimes name it `signal` and sometimes `abortSignal`. Based on the example it sounds like `signal` is the way to go.